src/screenbot.js

```diff
@@ -38,6 +38,21 @@ var Screenbot = (function Screenbot() {
       xhttp.send();
     };
 
+    var _handleError = function(args, cb){
+      var err = new ScreenbotError(args);
+      return cb(err);
+    };
+
+    var ScreenbotError = function(args){
```

Comment:
Could make the constructor arguments `(code, message)` then `message` would be optional and could auto-fill based on code if not specified.